src/main/java/com/assignment/application/test/CompanyServiceTest.java

```diff
@@ -0,0 +1,172 @@
+package com.assignment.application.test;
+
+
+import com.assignment.application.Constants.StringConstants;
+import com.assignment.application.controller.CompanyController;
+import com.assignment.application.entity.Company;
+import com.assignment.application.entity.CompleteCompInfo;
+import com.assignment.application.entity.Employee;
+import com.assignment.application.other.VerifyUser;
+import com.assignment.application.repo.CompanyRepo;
+import com.assignment.application.repo.EmployeeRepo;
+import com.assignment.application.service.CachingInfo;
+import com.assignment.application.service.CompanyServiceImpl;
+import com.assignment.application.service.interfaces.CompanyServiceI;
+import com.assignment.application.service.interfaces.EmployeeServiceI;
+import com.assignment.application.update.CompanyInfoUpdate;
+import org.hamcrest.Matchers;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CompanyServiceTest {
+
+    @Mock
+    private CompanyServiceImpl companyService;
+
+    @Mock
+    private CompanyRepo companyRepo;
+
+    @Mock
+    private CachingInfo cachingInfo;
+
+    @Mock
+    private StringConstants stringConstants;
```

Comment:
It is a static constant reference, no need to mock something where there is no behaviour to be mocked.